plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/table/actions/subscribe-button.tsx

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { MARKETPLACE_CART_PATH } from '~/marketplace/components/constants';
+import { appendURLParams } from '~/marketplace/utils/functions';
```

Comment:
The `~` looks weird, does it not mean the current user's home directory in this context? I see we do this also in `actions/connect-button.tsx`. It does seem to work.

Reply:
I left this by mistake, it's updated now.

`~` is the project home, in our case `plugins/woocommerce-admin/client`.